iterfzf/__init__.py

```diff
@@ -29,7 +29,7 @@ def iterfzf(
     # Search mode:
     extended=True, exact=False, case_sensitive=None,
     # Interface:
-    multi=False, mouse=True, print_query=False,
+    multi=None, mouse=True, print_query=False,
```

Comment:
We should change the type description in the *iterfzf/\_\_init\_\_.pyi* file.

https://github.com/dahlia/iterfzf/blob/2d13fdf04f2e85581feb721ace4064111e444cdf/iterfzf/__init__.pyi#L12